app/main/presenters/service_presenters.py

```diff
@@ -24,14 +24,15 @@ def _add_as_attribute_if_key_exists(self, key, service_data):
         if key[0] in service_data:
             setattr(self, key[1], service_data[key[0]])
 
-    def __init__(self, service_data, service_questions):
+    def __init__(self, service_data, service_questions, lots_by_slug):
         self.service_questions = service_questions
         # required attributes directly mapped to service_data values
         self.title = service_data['serviceName']
         self.serviceSummary = service_data['serviceSummary']\
             if 'serviceSummary' in service_data else \
             service_data['serviceDescription']
-        self.lot = service_data['lot']
+        # according to our test data, lot field in service_data may not be correct slug i.e. it may be mixed case
```

Comment:
The fixture in question is `dm-buyer-frontend/tests/fixtures/g6_service_fixture.json` and if you take out the `.lower()` backup call then the following tests fail:

`test_lot_attribute_is_set`
`test_g5_service_page_url`
`test_g6_service_page_url`

In the last two, it's assertions relating to the breadcrumbs that fail.